package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "oauth",
   "description": "Library for interacting with OAuth 1.0, 1.0A, 2 and Echo.  Provides simplified client access and allows for construction of more complex apis and OAuth providers.",
-  "version": "0.9.85",
+  "version": "0.9.86",
```

Comment:
why? 🤔

Reply:
When you change your lib you should update the version in order to work better with npm semver

Reply:
ok, didn't realize it was on another repo xD